plugins/python/profiler.c

```diff
@@ -13,6 +13,14 @@ int PyFrame_GetLineNumber(PyFrameObject *frame) {
 }
 #endif
 
+#if PY_VERSION_HEX < 0x030900B1
+PyCodeObject* PyFrame_GetCode(PyFrameObject *frame)
```

Comment:
Using https://github.com/python/pythoncapi-compat/ here would be interesting here, but I tried to mimick the coding style of the existing uwsgi code.

I can add PyFrame_GetLineNumber() to pythoncapi-compat if you want. Maybe using pythoncapi_compat.h can be done later.